_includes/head.html

```diff
@@ -20,7 +20,7 @@
       <link rel="stylesheet" href="{{ site.baseurl }}/css/content.css" />
       <link rel="stylesheet" href="{{ site.baseurl }}/css/post.css" />
       <link rel="stylesheet" href="{{ site.baseurl }}/css/mobile.css" />
-      <link href='http://fonts.googleapis.com/css?family=Open+Sans:700,300' rel='stylesheet' type='text/css'>
+      <link href='https://fonts.googleapis.com/css?family=Open+Sans:700,300' rel='stylesheet' type='text/css'>
```

Comment:
Just using //fonts.googleapis.com/css?family=Open+Sans:700,300 will help work with both http and https

Reply:
Sure I'll do that way